JustSaying/JustSayingFluently.cs

```diff
@@ -187,9 +187,20 @@ public ISubscriberIntoQueue WithSqsPointToPointSubscriber()
 
         #region Implementation of Queue Subscription
 
-        public IFluentSubscription IntoQueue(string queuename)
+        public IFluentSubscription IntoQueueNamed(string queueName)
         {
-            _subscriptionConfig.BaseQueueName = queuename;
+            if (string.IsNullOrWhiteSpace(queueName))
+            {
+                throw new ArgumentNullException(nameof(queueName));

```

Comment:
The intent is to reduce risk of accidentally getting 2 handlers on the same queue to a topic instead of 2 handlers each on their own queue  to a topic (or vice versa). I think that we will have to find other ways to do that, but still might keep `IntoDefaultQueue()` as an extension as you suggest.